doc/md/getting-started/dev-env.md

```diff
@@ -51,3 +51,37 @@ Before you start developing Motoko, verify the following:
 - [x] You have downloaded and installed [git](https://git-scm.com/downloads).
 
 - [x] Assure that all packages and tools above are updated to the latest release versions.
+
+## Motoko version
+
+Motoko version `0.11.1` is shipped with the latest version of the IC SDK (v0.20.0).
```

Comment:
Will we need to edit this every time we make a release? Maybe we can make it more generic.